assets/package.json

```diff
@@ -49,6 +49,7 @@
     "@reduxjs/toolkit": "^2.2.2",
     "axios": "^1.9.0",
     "axios-auth-refresh": "^3.3.6",
+    "canvas": "^3.1.2",
```

Comment:
`canvas` was apparently coming with other deps. It is not coming anymore with them, so I am adding it explicitly 🤷